### ISSUE TITLE
default column values are incorrectly handled by  diffs

### ISSUE THREAD

**Issue by edwardotis** [grails/gorm-hibernate5#978]
Given

```
class Apple{
Boolean isGood = false

    static mapping = {
        isGood defaultValue: false
    }
}
```
a. I have to hand correct the diff to add defaultvalueboolean:

`    changeSet(author: "me (generated)", id: "1486375845934-1") {
        addColumn(tableName: "apple") {
            column(name: "is_good", type: "boolean", defaultValueBoolean: false) {
                constraints(nullable: "false")
            }
        }
    }`

b. After applying update, the next diff will try to remove it

```
    changeSet(author: "me (generated)", id: "1486614694673-15") {
        dropDefaultValue(columnDataType: "boolean", columnName: "is_good", tableName: "apple")
    }
```

So basically, the plugin (or underlying libraries) break default values.

http://docs.grails.org/3.1.14/ref/Database%20Mapping/column.html

----
runtime 'org.grails.plugins:database-migration:2.0.0
grails 3.1.14
postgres 9.6
java 8
